building/userland_integration.rst

```diff
@@ -250,36 +252,46 @@ Then, we can ask OpenSSL to create a CSR from the key we have previously created
     engine "pkcs11" set.
 
 We can then inspect said CSR:
 
 .. code-block:: none
 
-    $ openssl req -in mykey_csr.pem -text
+    $ openssl req -in mykey_csr.pem -text -noout
     Certificate Request:
-        Data:
-            Version: 1 (0x0)
-            Subject: CN = My CSR
-            Subject Public Key Info:
-                Public Key Algorithm: id-ecPublicKey
-                    Public-Key: (256 bit)
-                    pub:
-                        04:e3:f8:9b:d3:2a:c8:10:1b:a6:75:81:5f:ba:f3:
-                        4c:4f:34:bb:7b:b2:d2:33:58:99:83:ba:d9:34:cf:
-                        a0:97:95:d5:68:11:74:77:78:d2:2b:94:e2:45:02:
-                        8d:3a:f6:af:f9:e6:ab:bb:db:3a:75:fe:14:33:18:
-                        2c:60:58:68:c7
-                    ASN1 OID: prime256v1
-                    NIST CURVE: P-256
-            Attributes:
-                a0:00
-        Signature Algorithm: ecdsa-with-SHA256
-             30:45:02:20:61:7e:05:30:cf:4d:d0:93:22:78:9e:45:cf:af:
-             3c:83:bb:04:c4:f0:81:f6:9a:5c:97:cd:ac:1e:94:cd:17:1b:
-             02:21:00:e7:7f:88:1d:4f:56:b8:e2:87:be:76:de:28:b3:92:
-             68:a7:16:3a:56:af:79:2f:98:bd:fd:6d:b3:82:e1:15:6c
+    Data:
+        Version: 1 (0x0)
+        Subject: CN = My CSR
+        Subject Public Key Info:
+            Public Key Algorithm: id-ecPublicKey
+                Public-Key: (521 bit)
+                pub:
+                    04:00:ca:d4:fe:70:d9:d1:44:e1:74:4e:da:55:45:
+                    a4:b1:03:ad:18:c4:38:51:5d:e8:06:cb:53:47:1d:
+                    26:6f:49:5b:58:ac:5d:30:90:8a:62:5a:e7:7f:f9:
+                    ed:cd:dc:4a:25:75:7c:7c:d0:b4:66:0d:c9:ea:c6:
+                    63:85:aa:2f:23:c9:21:01:77:8a:52:6a:6a:01:49:
+                    46:9b:18:75:47:aa:60:3c:2f:6d:8f:b4:93:82:6a:
+                    f2:e9:76:71:3c:57:56:ed:92:53:14:66:57:64:10:
+                    48:f3:4f:d1:93:07:ca:e7:3d:dc:3c:fd:0e:c6:79:
+                    35:62:4b:c1:43:54:1b:41:94:0c:39:1a:b4
+                ASN1 OID: secp521r1
+                NIST CURVE: P-521
+        Attributes:
+            (none)
+            Requested Extensions:
+    Signature Algorithm: ecdsa-with-SHA256
+    Signature Value:
+        30:81:88:02:42:01:e5:ce:68:9d:a3:59:84:1e:a4:aa:18:46:
+        03:c1:bb:ed:2e:ff:34:4a:78:97:6e:43:49:e0:5e:c8:15:6b:
+        5f:a1:c6:4c:95:3c:df:88:90:52:7f:07:59:e2:ec:9b:fb:51:
+        b0:3e:d0:0c:e6:de:8c:d6:56:c3:ac:3c:a6:9a:c1:0f:be:02:
+        42:01:78:04:b6:9d:25:c6:8b:fb:b5:87:26:de:84:c5:22:1c:
+        88:17:a0:ee:fb:1e:e7:e5:42:0e:5a:c7:6a:0d:3d:32:b9:a6:
+        3a:68:6f:ab:46:36:85:13:cf:84:94:4a:f4:21:19:26:84:30:
+        7d:cc:cb:4f:90:b2:99:32:5b:e7:8e:4d:dc
 
 Note that the public key matches exactly that which we have previously created
 (``04 e3 f8...``). This CSR could then be signed by a CA. For simplicity
```

Comment:
This comment is to be updated also.
I would expect the pub key at line 271 to start with `04:00:4d:1f:...`, to match the pub key value at line 197 (`EC_POINT: 04818504004d1f...` where prefix `048185` denotes an uncompressed long form 133 byte key)